php-migrations/Emergence/People/20191209_system-user.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Emergence\People;
+
+use DB;
+use SQL;
+
+
+function systemUserExists() {
+    return !!User::getByWhere(['Username' => 'system']);
+}
+
+$tableName = User::$tableName;
+
+if (!static::tableExists($tableName)) {
+    printf('Table `%s` does not exist, skipping.', $tableName);
```

Comment:
in the case of this migration, we don't want to skip if the table doesn't exist because unlike with schema changes the table won't start out in the right state when it gets created later.

Instead, create the table if it doesn't exist:

```php
DB::multiQuery(SQL::getCreateTable(User::class));
```